include/aikido/planner/kunzretimer/KunzRetimer.hpp

```diff
@@ -36,23 +39,42 @@ std::unique_ptr<aikido::trajectory::Spline> computeKunzTiming(
     const aikido::trajectory::Interpolated& inputTrajectory,
     const Eigen::VectorXd& maxVelocity,
     const Eigen::VectorXd& maxAcceleration,
-    double maxDeviation = 1e-2,
-    double timeStep = 0.1);
+    double maxDeviation = DEFAULT_MAX_DEVIATION,
+    double timeStep = DEFAULT_TIME_STEP);
 
 /// Class for performing time-optimal trajectory retiming following subject to
 /// velocity and acceleration limits.
 class KunzRetimer : public aikido::planner::TrajectoryPostProcessor
 {
 public:
+  /// Kunz postprocessor parameters used with the main `PostProcessorParams`
```

Comment:
Remove the `PostProcessorParams` mention.

Reply:
Whoops, done!